crates/stc_utils/src/lib.rs

```diff
@@ -1,8 +1,5 @@
 #![feature(never_type)]
 
-/// Use good memory allocator.
-extern crate swc_node_base;
```

Comment:
@kdy1 

I discovery this code is unused
So I think it's safe to delete it, what do you think?

> `swc` uses this when attempting to bind to the napi.rs.

Reply:
IIRC, `swc_node_base` is about using a faster mallocator